apps/els_lsp/test/els_completion_SUITE.erl

```diff
@@ -1675,44 +1675,49 @@ resolve_application_remote_otp(Config) ->
         <<"write/2">>
     ),
     #{result := Result} = els_client:completionitem_resolve(Selected),
-    Value =
-        case has_eep48(file) of
-            true ->
-                <<
-                    "```erlang\nwrite(IoDevice, Bytes) -> ok | {error, "
-                    "Reason}\nwhen\n  IoDevice :: io_device() | atom(),\n  Bytes ::"
-                    " iodata(),\n  Reason :: posix() | badarg | terminated.\n```\n\n"
-                    "---\n\nWrites `Bytes` to the file referenced by `IoDevice`\\. "
-                    "This function is the only way to write to a file opened in `raw`"
-                    " mode \\(although it works for normally opened files too\\)\\. "
-                    "Returns `ok` if successful, and `{error, Reason}` otherwise\\."
-                    "\n\nIf the file is opened with `encoding` set to something else "
-                    "than `latin1`, each byte written can result in many bytes being "
-                    "written to the file, as the byte range 0\\.\\.255 can represent "
-                    "anything between one and four bytes depending on value and UTF "
-                    "encoding type\\.\n\nTypical error reasons:\n\n* **`ebadf`**  \n"
-                    "  The file is not opened for writing\\.\n\n* **`enospc`**  \n"
-                    "  No space is left on the device\\.\n"
-                >>;
-            false ->
-                <<
-                    "## file:write/2\n\n---\n\n```erlang\n\n  write(File, "
-                    "Bytes) when is_pid(File) orelse is_atom(File)\n\n  write(#file_"
-                    "descriptor{module = Module} = Handle, Bytes) \n\n  write(_, _) "
-                    "\n\n```\n\n```erlang\n-spec write(IoDevice, Bytes) -> ok | "
-                    "{error, Reason} when\n      IoDevice :: io_device() | atom(),"
-                    "\n      Bytes :: iodata(),\n      Reason :: posix() | "
-                    "badarg | terminated.\n```"
-                >>
-        end,
-    Expected = Selected#{
-        documentation =>
-            #{
-                kind => <<"markdown">>,
-                value => Value
-            }
-    },
-    ?assertEqual(Expected, Result).
+    ?assertMatch(
+        #{
+            data := #{module := <<"file">>, function := <<"write">>, arity := 2},
+            documentation := #{kind := <<"markdown">>, value := _}
+        },
+        Result
+    ),
+    #{documentation := #{value := ActualValue}} = Result,
```

Comment:
This refactor was made to make case the last part of the expression, so it can return `{skip, _}`.

(The `data := #{module := <<"file">>, function := <<"write">>, arity := 2}` check is new though.)

Reply:
Ah, this is no longer useful after the latest few commits in this branch.